README.md

```diff
@@ -7,6 +7,8 @@
 
 <img src="./docs/images/kubeedge-logo-only.png">
 
+[![README](https://img.shields.io/badge/README-%E4%B8%AD%E6%96%87-brightgreen)](./README_zh.md)
```

Comment:
seems need to be moved to be before the kubeedge logo, put it together with other like `build status` and `releases`    and this img should be upload to the kubeedge path  like https://img.shields.io/github/license/kubeedge/kubeedge.svg?style=flat-square

Reply:
ok

Reply:
ok